AWS_Backup/AMI_Backup.py

```diff
@@ -0,0 +1,245 @@
+
```

Comment:
Remove the first empty line and make sure the file is renamed with lower case something like `ami_backup.py`

Reply:
Renamed the file